entities/utils/helpers.js

```diff
@@ -0,0 +1,58 @@
+const containsKeysOrVals = (value, target) => {
```

Comment:
Another non-crucial comment for v1 -- these types of simple functions might be prime candidates for unit testing, which would serve not only to validate their functionality with a variety of inputs, but also as documentation